cluster/node-pools/master-default/userdata.yaml

```diff
@@ -433,6 +433,51 @@ write_files:
           - mountPath: /etc/kubernetes/ssl
             name: ssl-certs-kubernetes
             readOnly: true
+{{ if eq .Cluster.ConfigItems.apiserver_internal_proxy "true" }}
+        - name: skipper-proxy-internal
+          image: registry.opensource.zalan.do/pathfinder/skipper:v0.11.35
+          args:
+          - skipper
+          - -access-log-strip-query
+          - -address=:443
+          - -support-listener=:9912
+          - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+          - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+          - -insecure
+          - -experimental-upgrade
+          - -runtime-metrics
+          - -enable-connection-metrics
+          - -enable-prometheus-metrics
+          - -write-timeout-server=60m
+          - -inline-routes
+          - |
+            s: JWTPayloadAllKV("kubernetes.io/serviceaccount/service-account.name", "default")
+              -> enableAccessLog()
```

Comment:
Maybe add the auditLog filter https://opensource.zalando.com/skipper/reference/filters/#auditlog to log data from the body in order to get more information about the client. I don't really know if this is valuable, but might be interesting to test.

Reply:
we could try

Reply:
Tried `auditlog` filter but it doesn't add additional information in our case.